packages/shared/src/styles/base.css

```
@@ -52,6 +52,7 @@ body {
   --theme-background-bun: theme('colors.bun.80');
   --theme-background-onion: theme('colors.onion.80');
   --theme-background-pepper: theme('colors.pepper.20');
+  --theme-background-cabbage-opacity-24: theme('colors.cabbage.40')24;
```

The hex value for 24% should be 3D, I think.

Reference: https://gist.github.com/lopspower/03fb1cc0ac9f32ef38f4

```suggestion
  --theme-background-cabbage-opacity-24: theme('colors.cabbage.40')3D;
```

---

Did not realize we are using hex values. Updated 👌

---

Me too, it was until @rebelchris gave me that link months ago 😂